packages/cubejs-cli/cubejsCli.js

```diff
@@ -11,6 +11,8 @@ const os = require('os');
 const chalk = require('chalk');
 const spawn = require('cross-spawn');
 const crypto = require('crypto');
+const inquirer = require('inquirer');
+const Drivers = require('@cubejs-backend/server-core/core/DriverDependencies');
```

Comment:
@paveltiunov There are two options related this moment:

1. Require `server-core` package
2. Create a yet another list of databases inside `cli` package?

What do you think?

Reply:
Btw:

Server-core requires version bump, before this PR will be landed, refs https://github.com/cube-js/cube.js/commit/558001841a97fea58913c302915fa21ae2a4ee97 (DriverDependencies was excluded to own file from `index.js`)